File: Aula 07/Exemplo 02.py
class BancoDeDadosLocal:
    def conectar(self):
        return 'Conectado ao DB interno.'

class AplicacaoWeb:
    def _init_ (self):
        self.db = BancoDeDadosLocal()

    def iniciar(self):
        print(self.db.conectar())

# Execução do Sistema
app = AplicacaoWeb()
# 0 app cria seu próprio banco de dados internamente.
# Deletar 'app' deleta o banco junto.

File: Aula 07/Exemplo 02.js
class BancoDeDadosLocal {
conectar() { return 'Conectado ao DB interno.'; }
}
class AplicacaoWeb {
    constructor() {
    this.db = new BancoDeDadosLocal();
    iniciar();
    console.log(this.db.conectar());
    }
}

// Execução do Sistema
const app = new AplicacaoWeb();
// A responsabilidade de instanciar o DB é exclusiva da Aplicação.